pkg/apis/configuration/validation/validation.go

```diff
@@ -857,11 +898,38 @@ func validateVirtualServerRouteHost(host string, virtualServerHost string, field
 	return allErrs
 }
 
+var regexOrExactMatchPrefixes = []string{"~", "="}
+
+func isRegexOrExactMatch(path string) bool {
+	for _, p := range regexOrExactMatchPrefixes {
+		if strings.HasPrefix(path, p) {
+			return true
+		}
+	}
+
+	return false
+}
+
 func validateVirtualServerRouteSubroutes(routes []v1alpha1.Route, fieldPath *field.Path, upstreamNames sets.String, pathPrefix string) field.ErrorList {
```

Comment:
should pathPrefix be renamed because it is no longer a prefix - could be regex or an exact match?
perhaps `vsPath` ?